src/models/Event.ts

```diff
@@ -137,6 +140,12 @@ const eventSchema = new Schema(
       ref: "User",
       required: true,
     },
+    actionItems: [
```

Comment:
get rid of this, relationship to an event is already present on `ActionItem` using `eventId` field